### PR TITLE
[Doc] Graph neural network and its variant Edit for grammar and style

### PR DESCRIPTION
**Pull request by john-andrilla**
Improve readability

## Description
Edit pass

### CODE REVIEW

**Comment by BarclayII** [comment]
Hi John,

Thanks for your edits!  Is it possible for you to merge all your editorial PRs into a single one?  That way we could manage all your changes more easily.  You could also make further editorial changes in that PR as well.

Thanks again.

**Comment by aaronmarkham** [comment]
> Hi John,
> 
> Thanks for your edits! Is it possible for you to merge all your editorial PRs into a single one? That way we could manage all your changes more easily. You could also make further editorial changes in that PR as well.
> 
> Thanks again.

Hi Andy, in tech writing we typically keep our changes discrete and PRs limited to subsections or else they get too big to review and hard to merge or deconflict. Since John is pretty new to open source, can we not overwhelm him with refactoring his pull requests? Also, when the PRs go out of date and require multiple rebases, it adds more difficulty to the process and inhibits newcomers' contributions to OSS. 

What can we do to expedite the process and make it easier for editors and tech writers?

Can you explain why you want to keep things in one PR instead of just merging the changes as they come in? 

I think for future changes we can put them into a single PR, but I'd like to know your thought process.

Thanks,
Aaron

**Comment by BarclayII** [comment]
Hi @aaronmarkham,

Thanks for your inputs.  What you and John are thinking matter a lot to us.

Our rationale from a CI standpoint are that (1) each PR would go through CI, which is slow and costly at the moment, and (2) merging one PR would need to update the other PRs, which in turn would require more builds on CI.  Because our documentations contain runnable code, the documentations also need CI testing to make sure nothing apparent is broken.

Another reason that I favor a single PR for documentation is that the eventual complete changes in the entire documentation would be more visible, especially since we wish to keep track of the context and organization of the documentation as a whole.  If I wish to read the version of the documentation with all the edits, I don’t have to merge all the PRs by myself and read the end product.

That being said, it is *completely okay to keep your edits in the current way* (that is, one change per PR).  Your reason also makes sense in the standpoint of technical editors, and we do not want to hinder your contributions.

To ease our management and your contribution, how about you keep going with small edits as separate PRs, and notify us (maybe with the correct order of reviewing the PRs, if they are not in order already) when you are done?

Best,

**Comment by BarclayII** [comment]
@aaronmarkham

Next time, I would suggest the following procedure to make documentation contributions:

1. Make a fork of the official repository on the GitHub web page (on the top-right corner you could see the button).
2. Make your changes in your own fork.  You can edit them as usual on the web interface.  However, as long as you are playing with your own fork, the changes will not reflect as PRs immediately.
3. After you are done with everything, make a pull request summarizing your changes.  You can also do this in the web interface.

Hope this helps.

**Comment by aaronmarkham** [comment]
> @aaronmarkham
> 
> Next time, I would suggest the following procedure to make documentation contributions (which I assume does not require any knowledge of Git CLI):
> 
> 1. Make a fork of the official repository on the GitHub web page (on the top-right corner you could see the button).
> 2. Make your changes in your own fork.  You can edit them as usual on the web interface.  However, as long as you are playing with your own fork, the changes will not reflect as PRs immediately.
> 3. After you are done with everything, make a pull request summarizing your changes.  You can also do this in the web interface.
> 
> Hope this helps.

I understand the CI issues! This sounds like a good plan.
Thanks for the feedback and clarifications.

**Comment by john-andrilla** [comment]
I understand the suggestions for next time and also the status quo current plan to keep going with small edits as separate PRs, and also notify you when I am done. Edits can be reviewed in the order created. To notify you, I will add @BarclayII @aaronmarkham to the PR comment. @mention other people too?

File: tutorials/models/1_gnn/README.txt [hunk 1]
```diff
@@ -1,48 +1,46 @@
 .. _tutorials1-index:
 
-Graph Neural Network and its variant
+Graph neural network and its variant
```
Comment (aaronmarkham) [inline tutorials/models/1_gnn/README.txt]:
```suggestion
Graph neural networks and its variants
```

File: tutorials/models/1_gnn/README.txt [hunk 2]
```diff
@@ -1,48 +1,46 @@
...
-  its own.
+  Relational-GCN allows multiple edges among two entities of a
+  graph. Edges with distinct relationships are encoded differently. 
 
-* **LGNN** `[paper] <https://arxiv.org/abs/1705.08415>`__ `[tutorial]
+* **Line graph neural network (LGNN)** `[research paper] <https://arxiv.org/abs/1705.08415>`__ `[tutorial]
   <1_gnn/6_line_graph.html>`__ `[Pytorch code]
   <https://github.com/dmlc/dgl/tree/master/examples/pytorch/line_graph>`__:
-  this model focuses on community detection by inspecting graph structures. It
+  This model focuses on community detection by inspecting graph structures. It
```
Comment (aaronmarkham) [inline tutorials/models/1_gnn/README.txt]:
```suggestion
  This network focuses on community detection by inspecting graph structures. It
```

**Pull request merged**